<launch>
  <node name="segmentation3d" pkg="shakey_object_recognition" type="objectDetection" output="screen">
    <rosparam file="$(find shakey_object_recognition)/config/config.yaml" command="load"/>
  </node>
</launch>
